indicators/migrations/0004_auto_20190201_1435.py

```diff
@@ -0,0 +1,70 @@
+# Generated by Django 2.1.5 on 2019-02-01 14:35
+
+import django.core.validators
+from django.db import migrations, models
+
+from indicators.utils import populate_ids_for_indicator
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('indicators', '0003_auto_20190126_1326'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='countryindicator',
+            name='IND',
+            field=models.PositiveIntegerField(validators=[django.core.validators.MinValueValidator(0), django.core.validators.MaxValueValidator(100)], verbose_name='IND'),
+        ),
+        migrations.RenameField('countryindicator', 'IND', 'ind'),
+        migrations.AlterField(
+            model_name='countryindicator',
+            name='IVR',
+            field=models.PositiveIntegerField(validators=[django.core.validators.MinValueValidator(0), django.core.validators.MaxValueValidator(100)], verbose_name='IVR'),
```

Comment:
According to [pep-8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) maximum line length should be limited to 79 characters